inc/wplf-form-validation.php

```diff
@@ -60,3 +60,54 @@ function wplf_validate_required_empty( $return ) {
 
   return $return;
 }
+
+/**
+ * Check that submission has only fields that are set in form
+ */
+add_filter( 'wplf_validate_submission', 'wplf_validate_additional_fields' );
+function wplf_validate_additional_fields( $return ) {
+  // skip this validation if submission has already failed
+  if ( ! $return->ok ) {
+    return $return;
+  }
+
+  // skip this validation if it is disabled with filter for this form
+  $form = get_post( intval( $_POST['_form_id'] ) );
+  // global disable
+  $disable_validation = apply_filters( 'wplf_disable_validate_additional_fields', $disable_validation );
+  // disable by form id
+  $disable_validation = apply_filters( "wplf_{$_POST['_form_id']}_disable_validate_additional_fields", $disable_validation );
+  // disable by form slug
+  $disable_validation = apply_filters( "wplf_{$form->post_name}_disable_validate_additional_fields", $disable_validation );
+  if ( $disable_validation ) {
```

Comment:
In 1.5 we're going to start saving the plugin version to post meta when the form is created. 

Something like 

```
$version_high_enough = version_compare( get_post_meta( $form->ID, '_wplf_plugin_version', true), '1.5.0', '>=' );
if  ( $disable_validation &&  $version_high_enough) {
```

should do the trick without breaking existing forms. Or similar, I'm open to suggestions.